packages/language-server/src/utils.ts

```diff
@@ -294,3 +294,14 @@ export function getAstroInstall(basePaths: string[]): AstroInstall | undefined {
 		},
 	};
 }
+
+export function getWorkspacePnpPath(workspacePath: string): string | null {
+	const possiblePath = resolve(workspacePath, '.pnp.cjs');
```

Comment:
This operation can throw an error too, we should move it inside the `try/catch` block

Reply:
Done, thanks!